### DISCUSSION TITLE
Is it possible to disable or hide the new lists feature?

### DISCUSSION THREAD

**Discussion by Phate6660** [community/community#10742] [Other Feature Feedback, Questions, & Ideas]
To be honest, I really don't like it at all.
On top of the fact that I won't ever see myself using this feature.
So I was just wondering if there is, or will be be, any official way to disable it.
Before I try my hand at undoing the feature locally.

**Comment by bailey** [comment]
Hi there, @Phate6660! No, it is not possible to disable or hide Lists. Can I ask where it gets in your way throughout GitHub?

**Reply by Phate6660** [reply]
Sorry for the late reply! Today has been pretty busy for me.
I wrote this question drunk, I should've been more clear and specified what was bothering me. My bad.
So a couple things:
- Since the lists feature was implemented, I can't seem to find a button or link anywhere for looking at the stargazers of a repo.
  + (I know I just have to add `/stargazers` to the repo URL, but I like having a quick and easily accessible button.)
  + Looking at stargazers feels like a major thing to me. It's a way for me find more developers with similar interests, and I've actually found quite a few really cool devs to follow just by looking at people who star my, or other peoples', repos.
- Sometimes features like this seem to be pushed or enabled out of nowhere, and when I get back on the site I notice the changes and it really starts to throw me off.
  + (Don't get me wrong, I'm sure in time I'll probably start using the feature and liking it. But I have issues and when I get put off like that I just immediately start disliking what that feature is.)

**Reply by SimonPilkington** [reply]
I would like to add on that currently the lists feature sits on top of the Stars tab, taking up space completely uselessly if the user isn't interested in using it, and impossible to hide. This feels like bad UX.

**Reply by bailey** [reply]
> Since the lists feature was implemented, I can't seem to find a button or link anywhere for looking at the stargazers of a repo.

We moved the link to view stargazers under the `About` section - hope this helps!

<img width="489" alt="Screen Shot 2021-12-09 at 3 45 02 PM" src="https://user-images.githubusercontent.com/6895176/152011757-46474e47-5287-4e9d-9add-8495a74312b4.png">

If you ever do decide to give Lists a try, let us know what you think! Hopefully the stargazers link is still easy to access for you in it's new home. Thanks again for all of this feedback. 😄

**Reply by ascherer** [reply]
> Can I ask where it gets in your way throughout GitHub?

I have starred fourty repos and can easily handle this single list. As long as I can't opt-out of this beta feature, I have to scroll oved the huge "start your first list" nuissance.

**Reply by trickyholley** [reply]
I personally would like an option to hide it or at _least_ move it to its own page apart from the stars. I personally want a page entirely dedicated to my starred repos for quick access to work items. As a result, I'm forced to either click an additional time (since lists apparently hide the contents until clicked on), or I have to bookmark inside the one list you're making me use to have my starred items at the top like they should've been in the first place (workaround to ignore lists anyhow).

I can see how lists can be useful, I don't have a problem innately with them - they just need to stay out of the way of my normal starred items 🙂

**Comment by ascherer** [comment]
For me it completely destroyed the stars page so I unstarred all projects.
